splash/qtrender_lua.py

```diff
@@ -174,7 +174,7 @@ class Splash(object):
     (wrapped in 'Splash' Lua object; see :file:`splash/lua_modules/splash.lua`).
     """
     _result_content_type = None
-    _attribute_whitelist = ['commands', 'args']
+    _attribute_whitelist = ['commands', 'args', 'set_js_enabled']
```

Comment:
Why not use `@command`? I don't like putting `set_js_enabled` to `_attribute_whitelist` because all other functions are declared as commands.